app/views/comments/_comment.html.erb

```diff
@@ -1,27 +1,56 @@
 <li class="comment">
-  <p class="author"><strong><%= comment.user.username %></strong></p>
-  <%= comment.body.html_safe %>
-  <% if current_user %>
-    <a class="create-link" href="javascript:void(0)" id="link-collapse-<%= comment.id %>" onclick="collapseReplyDiv('collapse-<%= comment.id %>')" style="font-size: 12px;">Reply</a>
-      <div class="form-comment" id="form-collapse-<%= comment.id %>" style="display: none">
-        <%= form_for :comment, url: comments_path, html: {class: 'new-form' } do |form| %>
-            <p class="form-p">
-              <%= form.label :body, 'Enter comment:' %>
-              <%= form.cktext_area :body, class: 'textarea-field', id: "comment_#{comment.id}", ckeditor: { uiColor: '#ffe6bd', toolbar: 'mini', height: 100, width: 400 } %>
-            </p>
-        
-            <%= form.hidden_field :commentable_type, value: "Comment" %>
-            <%= form.hidden_field :commentable_id, value: comment.id %>
-            <%= form.hidden_field :parent_article, value: @article.id %>
+  <div class="comment-vote">
+    <% if current_user %>
+      <% if comment.votes.where(user: current_user).any? %>
+        <%= up_vote_links("Comment", comment.id, :put) %>
+        <div id="vote-Comment-count-<%= comment.id %>" class="comment-vote-count"><%= comment.votes.sum(:value) %></div>
+        <%= down_vote_links("Comment", comment.id , :put) %>
+      <% else %>
+        <%= up_vote_links("Comment", comment.id , :post) %>
+        <div id="vote-Comment-count-<%= comment.id %>" class="comment-vote-count"><%= comment.votes.sum(:value) %></div>
+        <%= down_vote_links("Comment", comment.id , :post) %>
+      <% end %>
+    <% else %>
+      <%= link_to("", nil, id: "vote-comment-up-nil-#{comment.id}", class: "comment-vote-top") %>
+      <div id="vote-count-<%= comment.id %>" class="comment-vote-count"><%= comment.votes.sum(:value)  %></div>
+      <%= link_to("", nil , id: "vote-comment-down-nil-#{comment.id}", class: "comment-vote-bottom") %> 
+    <% end %>
+    
+    <script type="text/javascript">
+      $('#vote-Comment-button-up-<%= comment.id %>').bind('ajax:success', function(evt, data, status, xhr) {
+        changeVoteCount(evt, data, status,xhr, "Comment");
+      });
 
-            <p class="form-p">
-              <%= form.submit 'Reply', class: 'comment-reply' %>
-              <a class="create-link" id="" onclick="revCollapseReplyDiv('collapse-<%= comment.id %>')" style="font-size: 12px;">Back</a>
-            </p>
-        <% end %>
-       
-      </div>
-  <% end %>
+      $('#vote-Comment-button-down-<%= comment.id %>').bind('ajax:success', function(evt, data, status, xhr) {
+        changeVoteCount(evt, data, status, xhr, "Comment");
+      });
+    </script>
+  </div>
+  
+  <div class="comment-content">
+    <p class="author"><strong><%= comment.user.username %></strong></p>
```

Comment:
I'd suggest adjusting `.author` to bold the username instead of inline `<strong>` here